### ISSUE TITLE
Automatically apply the script on selected binaries

### ISSUE THREAD

**Issue by valour01** [NationalSecurityAgency/ghidra#268]
Hi. Thank you very much for releasing such a great tool. I have a question on the features of scripting. 

Suppose I have already write a script in python to do some analysis. I have several different kinds of binaries. I want to apply the script on these binaries automatically rather than following the steps to double click Ghidra, import the files, open the script manager and run the script. 

In short, what I need is a command like tools that could load the program into ghidra automatically and then run the specified scripts. I am not sure whether ghidra provide such tools or do you have any suggestions? Many Thanks

**Comment by saruman9** [comment]
See `${GHIDRA_HOME}/docs/GhidraClass/Intermediate/HeadlessAnalyzer_withNotes.html`.
